docker/ci/dockerfiles/current/build.centos7.opensearch-dashboards.x64.arm64.dockerfile

```diff
@@ -16,11 +16,6 @@ FROM centos:7
 # Ensure localedef running correct with root permission
 USER 0
 
-# Setup ENV to prevent ASCII data issues with Python3
```

Comment:
Is this no more an issue in 3.9?

Reply:
No, centos7 only has this issue on 3.7 due to glibc not 2.23 version.
Adding this would only show even more errors on compiling 3.9.

Removing and let everyone use default locale is the best case here, see description.